administration/src/bp-modules/statistics/components/StatisticsFilterBar.tsx

```diff
@@ -0,0 +1,102 @@
+import { Button, FormGroup, Tooltip } from '@blueprintjs/core'
+import { TextField } from '@mui/material'
+import React, { ReactElement, useState } from 'react'
+import styled from 'styled-components'
+
+import PlainDate from '../../../util/PlainDate'
+import StickyBottomBar from '../../StickyBottomBar'
+import { defaultEndDate, defaultStartDate } from '../constants'
+
+const StyledFormGroup = styled(FormGroup)`
+  margin: 0 16px 0 0;
+  align-self: center;
+`
+
+const InputContainer = styled.div`
+  flex-direction: row;
+  display: flex;
+  justify-content: flex-start;
+  flex: 1;
+  padding: 0 20px;
+`
+type StatisticsFilterBarProps = {
+  onApplyFilter: (dateStart: string, dateEnd: string) => void
+}
+
+const isValidDateString = (value: string): boolean => {
+  if (value !== null) {
+    try {
+      PlainDate.from(value)
+      return true
+    } catch (error) {
+      console.error(`Could not parse date from string '${value}'.`, error)
+      return false
+    }
+  }
+  return false
+}
+const IsValidDateTimePeriod = (dateStart: string, dateEnd: string): boolean => {
+  if (!isValidDateString(dateStart) || !isValidDateString(dateEnd)) {
+    return false
+  }
+  return PlainDate.compare(PlainDate.from(dateStart), PlainDate.from(dateEnd)) === -1
```

Comment:
```suggestion
  return PlainDate.compare(PlainDate.from(dateStart), PlainDate.from(dateEnd)) <= 0
```

Otherwise it's not possible to set a filter from today to today

Reply:
from today to today for me is no valid time period. The tooltip say the startDate has to be before the endDate

Reply:
Why we don't want to show the statistics for one day only?